<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>167</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>303</width>
    <height>579</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Select Algorithm</string>
  </property>
  <widget class="QPushButton" name="define_algo_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-image: url(:/stripes/bg/stripes-light.png);
	color: rgb(72, 61, 65);
	border-radius: 15px;
border-width: 2px;
border-style: solid;
	border-color: rgb(72, 61, 65);
}
</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string>Define Algorithm Parameters</string>
   </property>
   <property name="icon">
    <iconset resource="ui_assets/resources.qrc">
     <normaloff>:/icons/compute.png</normaloff>:/icons/compute.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/stripes/bg/stripes-light.png);</string>
   </property>
   <widget class="QComboBox" name="algorithm_combo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>Basic Trade</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Moving Average Crossover</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Let's Gamble</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pair Spread</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sentiment Bet</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ui_assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
